src/main/java/edu/ucsb/cs56/mapache_search/membership/AuthControllerAdvice.java

```diff
@@ -27,12 +27,39 @@ public boolean getIsLoggedIn(OAuth2AuthenticationToken token) {
         return token != null;
     }
 
+    //returns -1 for not logged in, returns 0 for github, 1 for google
+    @ModelAttribute("isGoogleOrGithub")
```

Comment:
I think this should be split into two boolean functions - isGoogle and isGithub. Otherwise, it'll be hard to understand what this function does.